scripts/sct_download_data.py

```diff
@@ -127,24 +128,46 @@ def main(args=None):
     data_name = arguments['-d']
     verbose = int(arguments.get('-v'))
     sct.init_sct(log_level=verbose, update=True)  # Update log level
-    dest_folder = arguments.get('-o', os.path.abspath(os.curdir))
+    dest_tmp_folder = arguments.get('-o', os.path.abspath(os.curdir)+'/tmp')
 
     # Download data
     url = dict_url[data_name]
     tmp_file = download_data(url, verbose)
 
+    # unzip
+    unzip(tmp_file, dest_tmp_folder, verbose)
+    data_extracted_name = data_name
+    if dest_tmp_folder != os.path.abspath(os.curdir)+'/bin':
```

Comment:
why "bin"?

Reply:
Because it is the path where the installation requirements are downloaded.

Reply:
`sct_download_data` is not only used for installing binaries (look at the help)